BLMM/lib/fileio.py

```diff
@@ -517,3 +520,91 @@ def pracNumVoxelBlocks(inputs):
 
   # Return number of voxel blocks
   return(nvb)
+
+def cluster_detection(clustertype,maxmem):
+        # --------------------------------------------------------------------------------
+        # Set up cluster
+        # --------------------------------------------------------------------------------
+    with open(os.path.join(os.path.expanduser("~"), ".config/dask/jobqueue.yaml"), "r") as stream:
+        data = yaml.load(stream, Loader=yaml.FullLoader)
+    if maxmem < 21474836480:
```

Comment:
I found a quite a big problem with the memory usage so I hardcoded this minimum for the jobs, can be reviewed

Reply:
Hmm, we should discuss this issue further. I should probably ask a few questions to assess this:

 - First, could I ask where the number `21474836480` came from? 
 - When you experienced this error, was `MAXMEM` set to the default in the yaml file? If not, would the previous comment about MAXMEM fix this?
 - Was your issue to do with the working memory of the code? Or was it the disk space used by outputting files? There is a trade-off here; to avoid loading too much into memory during computation BLMM must temporarily save a lot to disk in the batch stage; could it be that you did not have much free memory in the output directory (which is where BLMM temporarily saves files during computations)?